backend/core/domains/payments/services/gateway_monitoring_service.py

```
@@ -0,0 +1,606 @@
+            return GatewayHealthCheck(
+                gateway_code=gateway_code,
+                is_healthy=False,
+                error_message=str(e),
+                consecutive_failures=cls._get_consecutive_failures(gateway_code) + 1
+            )
+
+    @classmethod
+    def _update_failure_tracking(cls, health_check: GatewayHealthCheck):
+        """Update failure tracking for gateway"""
+        gateway_code = health_check.gateway_code
+        tracking_key = f"gateway_failures_{gateway_code}"
```

Ruff (0.13.3)

Local variable `tracking_key` is assigned to but never used

Remove assignment to unused variable `tracking_key`

(F841)